src/components/raycaster.js

```diff
@@ -27,7 +27,8 @@ module.exports.Component = registerComponent('raycaster', {
     objects: {default: ''},
     origin: {type: 'vec3'},
     recursive: {default: true},
-    showLine: {default: false}
+    showLine: {default: false},
+    worldCoordinates: {default: false}
```

Comment:
I'd name this `isWorldSpace` (or `areWorldCoordinates`) to not sound like it takes coordinates.